ports/stm32/boards/manifest.py

```diff
@@ -2,3 +2,4 @@
 freeze("$(MPY_DIR)/drivers/dht", "dht.py")
 freeze("$(MPY_DIR)/drivers/display", ("lcd160cr.py", "lcd160cr_test.py"))
 freeze("$(MPY_DIR)/drivers/onewire", "onewire.py")
+include("$(MPY_DIR)/drivers/neopixel/manifest.py")
```

Comment:
I'm on the fence with adding this to stm32: it is convenient, but stm32 has not needed it so far and it's easy enough to just copy across `neopixel.py` if needed, and including it by default sets a precedence and makes it harder to remove from the build in the future.

Reply:
Yes, that makes sense. Happy to leave it out on STM32 for now.

Speaking of STM32, the CI failed for the NUCLEO_L073RZ board due to flash size (it was fine earlier in the week but this and fstrings pushes it over the limit). I've disabled machine.bitstream on that board. (FWIW I didn't notice this locally because later version of arm-none-eabi-gcc is a few hundred bytes better at size).

Reply:
Updated the branch with neopixel.py no longer added in ports/stm32/boards/manifest.py